### PR TITLE
Ipfs 0.4.16 release candidate 2

### PR DESCRIPTION
**Pull request by whyrusleeping**
License: MIT
Signed-off-by: Whyrusleeping <why@ipfs.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,63 @@
...
+default build. With this, ipfs can ingest git repositories and other data
+directly, and operate over it. For more information on this, see [the
+go-ipld-git repo](https://github.com/ipfs/go-ipld-git).
+
+Various other changes were merged in this release, including great
+documentation, a good number of smaller bugfixes, refactoring and a good bit
+more. For the details, see the changelog below.
+
+
+## 0.4.16-rc2 2018-07-05
```
Comment (Stebalien) [inline CHANGELOG.md]: Shouldn't this be a new section?

Comment (whyrusleeping) [reply CHANGELOG.md]: hrm... yeah i'm not sure what reads best here. 

When we do the actual release, would we have an empty section that just says there was not any changes between rc2 and the release?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,70 @@
...
+- Bugfixes
+  - Fix usage of file name vs path name in adder ([ipfs/go-ipfs#5167](https://github.com/ipfs/go-ipfs/pull/5167))
+  - Fix `ipfs update` working with migrations ([ipfs/go-ipfs#5194](https://github.com/ipfs/go-ipfs/pull/5194))
+- Documentation
+  - Grammer fix in fuse docs ([ipfs/go-ipfs#5164](https://github.com/ipfs/go-ipfs/pull/5164))
+
+## 0.4.16-rc2 2018-06-27
```
Comment (magik6k) [inline CHANGELOG.md]: Should be `0.4.16-rc1` here

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,70 @@
 # go-ipfs changelog
 
-## 0.4.16-rc1 2018-06-27
```
Comment (magik6k) [inline CHANGELOG.md]: Header missing?

Comment (whyrusleeping) [reply CHANGELOG.md]: Yeah, i'm really not sure how to format this anymore.

**Comment by whyrusleeping** [comment]
@Stebalien @magik6k take a look at the formatting now, maybe that looks better?

**Pull request merged**